Derivs/src/derivs.cxx

```diff
@@ -1,150 +0,0 @@
-#include "derivs.hxx"
-
-namespace Derivs {
-using namespace Arith;
-using namespace Loop;
-
-////////////////////////////////////////////////////////////////////////////////
-
-// Tile-based multi-dimensional derivative operators
-
-template <int CI, int CJ, int CK, typename T>
```

Comment:
The tile-based derivative operators in thorn `Derivs` were implemented in C++ sources files (as opposed to header files) because they would not be inlined. There is no advantage to having them in a header file, and one disadvantage is that it increased build times.